articles/backup/backup-azure-vms-automation.md

```diff
@@ -523,6 +523,47 @@ A user can selectively restore few disks instead of the entire backed up set. Pr
 
 Once you restore the disks, go to the next section to create the VM.
 
+#### Restore disks to Secondary region
+
+If CRR is enabled on the vault with which you have protected your VMs, the backup data is replicated to the secondary region and hence can be used to perform a restore. Perform the below steps to trigger restore in secondary region:
```

Comment:
If CRR is enabled on the vault with which you have protected your VMs, the backup data is replicated to the secondary region and so can be used to perform a restore. Perform the following steps to trigger a restore in the secondary region: